### PR TITLE
Cld 9211 add wait until to wdio wrapper

### PR DESCRIPTION
**Pull request by Ad1B3nAr0ya**

### CODE REVIEW

File: src/commons/BrowserUtils.ts [hunk 1]
```diff
@@ -209,6 +209,23 @@ export namespace BrowserUtils {
...
   }
 
+  /**
+   *  Wait Until - Will Return true in case condition met within the timeout or false if condition isn't met or not met within the timeout
+   * @param action - any condition as a function
+   * @param timeout - specified time out if undefined Default time out is used
+   */
+  export function waitUntil(
+    action: Function,
+    errMessage?: string,
```
Comment (FelixZilber) [inline src/commons/BrowserUtils.ts]: add errMessage to jdocs params

File: test/specs/WaitUntilSpec.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
+// tslint:disable-next-line:no-http-string
```
Comment (FelixZilber) [inline test/specs/WaitUntilSpec.ts]: this one redundant

Comment (Ad1B3nAr0ya) [reply test/specs/WaitUntilSpec.ts]: its not.

Comment (Ad1B3nAr0ya) [reply test/specs/WaitUntilSpec.ts]: removed

**Pull request merged**